include/freerdp/server/audin.h

```diff
@@ -159,7 +159,7 @@ extern "C"
 	/** \brief sets the supported audio formats for AUDIN server channel context.
 	 *
 	 *  \param context The context to set the formats for
-	 *  \param count The number of formats found in \formats. Use \b -1 to set to default formats
+	 *  \param count The number of formats found in \b formats. Use \b -1 to set to default formats
```

Comment:
If I compare this with the next line, shouldn't this be `\param`, instead of `\b`?

Reply:
no, `\param` is for documenting a parameter (the name that follows) and `\b` is a formatting option